### PR TITLE
Create new Record class to store the past records of a patient's Height and Weight

### LINKED ISSUE #100: Create Record class to store past records of patient's height and weight
Related to #98

### PR DESCRIPTION
**Pull request by Wong-ZZ**
Resolves #100

### CODE REVIEW

File: src/main/java/seedu/medibook/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -72,4 +78,43 @@ public static ReadOnlyMediBook getSampleMediBook() {
...
+                date = date.minusDays((int) (Math.random() * 20));
+                int updatedHeight = Integer.parseInt(currentHeight) - (int) (Math.random() * 1.5);
+                currentHeight = Integer.toString(updatedHeight);
+
+                double updatedWeight = Double.parseDouble(currentWeight) + (weightVariance * (Math.random() - 0.5));
+                currentWeight = toOneDecimal(Double.toString(updatedWeight));
+            }
+        }
+        return patients;
+    }
```
Comment (divakarmal) [inline src/main/java/seedu/medibook/model/util/SampleDataUtil.java:105]: This is just to create sample records for testing?

File: src/test/java/seedu/medibook/storage/JsonAdaptedRecordTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+import org.junit.jupiter.api.Test;
+
+import seedu.medibook.commons.exceptions.IllegalValueException;
+import seedu.medibook.model.Date;
+import seedu.medibook.model.patient.Height;
+import seedu.medibook.model.patient.Record;
+import seedu.medibook.model.patient.Weight;
+
+public class JsonAdaptedRecordTest {
```
Comment (divakarmal) [inline src/test/java/seedu/medibook/storage/JsonAdaptedRecordTest.java:18]: Good testing!

**Review by divakarmal — APPROVED** [review]
Good implementation of records, comprehensive testing. lgtm

File: src/main/java/seedu/medibook/model/util/SampleDataUtil.java (hunk 1 above)
Comment (Wong-ZZ) [reply src/main/java/seedu/medibook/model/util/SampleDataUtil.java:105]: Yep

**Pull request merged**